pages/linux/rmlint.md

```diff
@@ -0,0 +1,20 @@
+# rmlint
+
+> Extremely fast tool to remove remove duplicates and other lint from your filesystem.
+> More information: <https://rmlint.readthedocs.io/en/latest/index.html>.
+
+- Check directory for duplicates, empty files, and other lint:
+
+`rmlint {{path/to/directory}}`
+
+- Delete lint files found by an rmlint run:
+
+`./rmlint.sh`
+
+- Find all duplicates with the same base name:
+
+`rmlint --match-basename {{path/to/directory}}`
+
+- Find all duplicates with the same extension:
+
+`rmlint --match-extension {{path/to/directory}}`
```

Comment:
I think this may be a little redundant with the previous example. How about an example for locating duplicate directories, which appears to be a unique feature of this tool among other duplicate finder tools?

Reply:
I'll get right to that:
- add `-S`
- add `-D`

Reply:
For future reference, these were added in [`cd1db75`](https://github.com/tldr-pages/tldr/pull/9090/commits/cd1db75628e0fa93d84d681769ec83ff5904dbc0).